### PR TITLE
Added average rating and reimplemented sidebar in mobile view

### LINKED ISSUE #3057: Filter by avg. customer rating not available in mobile view
**Actual Behaviour**

Currently filtering by the avg. user rating is not available in mobile views. 

**Expected Behaviour**

All forms of filtering available in the web view must be available in the mobile view.

**Screenshots of the issue**

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by anishagg17**
Fixes #3057

Changes:

Added average rating in mobile view

Demo Link: https://pr-3224-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 

Hnet com-image

### CODE REVIEW

**Comment by anishagg17** [comment]
Sir I have added all skill refiners in sidebar but there were some cors issue due to which none of the skillName was fetched and hence the list contains no categories and on the homepage no skill is found

**Comment by anishagg17** [comment]
@snitin315    please review

**Comment by snitin315** [comment]
> @snitin315 please review
Resolve the conflicts.

**Comment by anishagg17** [comment]
> @anishagg17 Resolve the conflicts.

Done Sir!

**Comment by anishagg17** [comment]
@akshatnitd please review this pr

**Comment by anishagg17** [comment]
> Also, do not remove the categories present at the bottom of the page.

Do you want drawer to have only refine by ratings?

**Comment by anishagg17** [comment]
Hnet com-image (4)


Is this fine?

**Comment by anishagg17** [comment]
Sir changes have been done

File: src/components/cms/BrowseSkill/BrowseSkill.js [hunk 1]
```diff
@@ -609,18 +611,18 @@ class BrowseSkill extends React.Component {
...
-        return (
-          <Link to={linkValue} key={linkValue}>
-            <SidebarItem key={categoryName} value={categoryName}>
-              {categoryName}
-            </SidebarItem>
-          </Link>
-        );
-      });
-    }
+    // if (!isMobile) {
```
Comment (akshatnitd) [inline src/components/cms/BrowseSkill/BrowseSkill.js]: Remove comments

Comment (anishagg17) [reply src/components/cms/BrowseSkill/BrowseSkill.js]: Done sir!

**Comment by anishagg17** [comment]
Done Sir! Changes have been done

**Pull request merged**